plugins/vst_base/RemoteVstPlugin32.cmake

```diff
@@ -39,6 +42,7 @@ ELSEIF(LMMS_BUILD_WIN64 AND MSVC)
 	ExternalProject_Add(RemoteVstPlugin32
 		"${EXTERNALPROJECT_ARGS}"
 		CMAKE_GENERATOR "${CMAKE_GENERATOR_32}"
+		CMAKE_GENERATOR_PLATFORM Win32
```

Comment:
Is that needed? I think it is the default.

Reply:
From the same link:
> For VS 2019 and above this is based on the host platform.

Reply:
Missed that :D

Reply:
From that link,
> For VS 2019 and above this is based on the host platform.

So for VS 2019 on a 64-bit machine, it will try to build a 64-bit RemoteVstPlugin32 without that line there.